Makefile

```diff
@@ -6,57 +6,89 @@ EXTERNAL_TOOLS=\
 	github.com/kardianos/govendor
 BUILD_TAGS?=${TOOL}
 GOFMT_FILES?=$$(find . -name '*.go' | grep -v vendor)
-# bin generates the releaseable binaries for this plugin
+
+PLUGIN_NAME?=$(shell command ls bin/)
+PLUGIN_DIR?=$$GOPATH/vault-plugins
+PLUGIN_MOUNT?=local-gcp
```

Comment:
Nit: suggest to change to `PLUGIN_PATH` to match the Vault CLI and docs description.